### ISSUE TITLE
NTR: C1s low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8335]
New term request C1s low in blood (elthat_130426114839):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000242
- label: Decreased circulating complement component 1s concentration
- parents: [Complement deficiency (HP:0004431)](https://hpo.jax.org/app/browse/term/HP:0004431)
- pmid: 
- def: The concentration of complement component 1s (C1s) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8335
- synonyms: C1s low in blood(NONE)
